src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task06.java

```diff
@@ -39,8 +39,17 @@ public static void main(String[] args) {
      * @return 'Player 1: ${0}. Player 2: ${1}', где ${0} - очки первого игрока, ${1} - очки второго игрока.
      */
     static String getResult(int tanksKilledFirstPlayer, int tanksKilledSecondPlayer) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        int resultPointsFirstPlayer = tanksKilledFirstPlayer * KILL_POINT;
+        int resultPointsSecondPlayer = tanksKilledSecondPlayer * KILL_POINT;
+
+        if (resultPointsFirstPlayer > resultPointsSecondPlayer) {
+            resultPointsFirstPlayer = resultPointsFirstPlayer + BONUS_POINT;
+        }
+        else if (resultPointsFirstPlayer < resultPointsSecondPlayer) {
+            resultPointsSecondPlayer = resultPointsSecondPlayer + BONUS_POINT;
+        }
+        else if (resultPointsFirstPlayer == resultPointsSecondPlayer) {
+            return "No bonus points " + "Player 1:" + resultPointsFirstPlayer + ". " + "Player 2:" + resultPointsSecondPlayer;}
```

Comment:
"Violation: There isn't separating white spaces (OR line break) for curly brake
Solution: Add white spaces"